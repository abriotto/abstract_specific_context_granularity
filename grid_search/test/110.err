  0%|                                                                                                                                                                                                    | 0/99 [00:00<?, ?it/s]  1%|#8                                                                                                                                                                                          | 1/99 [00:00<00:11,  8.84it/s]  2%|###7                                                                                                                                                                                        | 2/99 [00:00<00:12,  7.96it/s]  4%|#######5                                                                                                                                                                                    | 4/99 [00:00<00:10,  9.43it/s]  6%|###########3                                                                                                                                                                                | 6/99 [00:00<00:08, 10.60it/s]  8%|###############1                                                                                                                                                                            | 8/99 [00:00<00:07, 12.74it/s] 10%|##################8                                                                                                                                                                        | 10/99 [00:00<00:06, 12.82it/s] 12%|######################6                                                                                                                                                                    | 12/99 [00:01<00:08, 10.21it/s] 15%|############################3                                                                                                                                                              | 15/99 [00:01<00:06, 12.19it/s] 17%|################################1                                                                                                                                                          | 17/99 [00:01<00:07, 11.69it/s] 19%|###################################8                                                                                                                                                       | 19/99 [00:01<00:07, 11.40it/s] 21%|#######################################6                                                                                                                                                   | 21/99 [00:01<00:06, 11.15it/s] 23%|###########################################4                                                                                                                                               | 23/99 [00:02<00:06, 11.58it/s] 25%|###############################################2                                                                                                                                           | 25/99 [00:02<00:06, 12.30it/s] 27%|##################################################9                                                                                                                                        | 27/99 [00:02<00:05, 12.14it/s] 29%|######################################################7                                                                                                                                    | 29/99 [00:02<00:06, 10.01it/s] 31%|##########################################################5                                                                                                                                | 31/99 [00:02<00:06, 11.25it/s] 33%|##############################################################3                                                                                                                            | 33/99 [00:03<00:07,  9.25it/s] 35%|##################################################################1                                                                                                                        | 35/99 [00:03<00:06,  9.34it/s] 37%|#####################################################################8                                                                                                                     | 37/99 [00:03<00:05, 10.37it/s] 39%|#########################################################################6                                                                                                                 | 39/99 [00:03<00:06,  9.80it/s] 41%|#############################################################################4                                                                                                             | 41/99 [00:03<00:06,  9.16it/s] 42%|###############################################################################3                                                                                                           | 42/99 [00:04<00:06,  8.40it/s] 45%|#####################################################################################                                                                                                      | 45/99 [00:04<00:04, 11.57it/s] 47%|########################################################################################7                                                                                                  | 47/99 [00:04<00:04, 10.47it/s] 49%|############################################################################################5                                                                                              | 49/99 [00:04<00:04, 12.07it/s] 52%|################################################################################################3                                                                                          | 51/99 [00:04<00:04, 10.70it/s] 54%|####################################################################################################1                                                                                      | 53/99 [00:05<00:04,  9.80it/s] 56%|#######################################################################################################8                                                                                   | 55/99 [00:05<00:04,  9.47it/s] 58%|###########################################################################################################6                                                                               | 57/99 [00:05<00:04,  9.02it/s] 59%|#############################################################################################################5                                                                             | 58/99 [00:05<00:04,  8.44it/s] 60%|###############################################################################################################4                                                                           | 59/99 [00:05<00:05,  7.73it/s] 61%|#################################################################################################################3                                                                         | 60/99 [00:05<00:05,  7.38it/s] 62%|###################################################################################################################2                                                                       | 61/99 [00:06<00:05,  7.15it/s] 63%|#####################################################################################################################1                                                                     | 62/99 [00:06<00:05,  6.56it/s] 65%|########################################################################################################################8                                                                  | 64/99 [00:06<00:03,  8.99it/s] 67%|############################################################################################################################6                                                              | 66/99 [00:06<00:03,  8.61it/s] 68%|##############################################################################################################################5                                                            | 67/99 [00:06<00:04,  7.79it/s] 69%|################################################################################################################################4                                                          | 68/99 [00:07<00:04,  7.41it/s] 70%|##################################################################################################################################3                                                        | 69/99 [00:07<00:04,  6.97it/s] 71%|####################################################################################################################################2                                                      | 70/99 [00:07<00:04,  6.61it/s] 72%|######################################################################################################################################1                                                    | 71/99 [00:07<00:04,  6.55it/s] 73%|########################################################################################################################################                                                   | 72/99 [00:07<00:04,  6.35it/s] 74%|#########################################################################################################################################8                                                 | 73/99 [00:07<00:04,  6.18it/s] 75%|###########################################################################################################################################7                                               | 74/99 [00:07<00:04,  6.23it/s] 76%|#############################################################################################################################################6                                             | 75/99 [00:08<00:03,  6.27it/s] 78%|#################################################################################################################################################4                                         | 77/99 [00:08<00:03,  7.10it/s] 80%|#####################################################################################################################################################2                                     | 79/99 [00:08<00:02,  7.90it/s] 81%|#######################################################################################################################################################1                                   | 80/99 [00:08<00:02,  7.41it/s] 83%|##########################################################################################################################################################8                                | 82/99 [00:09<00:02,  7.61it/s] 84%|############################################################################################################################################################7                              | 83/99 [00:09<00:02,  7.27it/s] 86%|################################################################################################################################################################5                          | 85/99 [00:09<00:01,  7.58it/s] 87%|##################################################################################################################################################################4                        | 86/99 [00:09<00:01,  7.11it/s] 88%|####################################################################################################################################################################3                      | 87/99 [00:09<00:01,  6.61it/s] 89%|######################################################################################################################################################################2                    | 88/99 [00:09<00:01,  6.38it/s] 91%|##########################################################################################################################################################################                 | 90/99 [00:10<00:01,  8.68it/s] 93%|#############################################################################################################################################################################7             | 92/99 [00:10<00:00,  8.52it/s] 94%|###############################################################################################################################################################################6           | 93/99 [00:10<00:00,  7.95it/s] 95%|#################################################################################################################################################################################5         | 94/99 [00:10<00:00,  7.28it/s] 96%|###################################################################################################################################################################################4       | 95/99 [00:10<00:00,  6.91it/s] 98%|#######################################################################################################################################################################################2   | 97/99 [00:10<00:00,  8.74it/s] 99%|#########################################################################################################################################################################################1 | 98/99 [00:11<00:00,  7.85it/s]100%|###########################################################################################################################################################################################| 99/99 [00:11<00:00,  7.42it/s]100%|###########################################################################################################################################################################################| 99/99 [00:11<00:00,  8.77it/s]
  0%|                                                                                                                                                                                                    | 0/25 [00:00<?, ?it/s]  8%|###############                                                                                                                                                                             | 2/25 [00:00<00:02,  8.14it/s] 16%|##############################                                                                                                                                                              | 4/25 [00:00<00:01, 10.78it/s] 24%|#############################################1                                                                                                                                              | 6/25 [00:00<00:01,  9.58it/s] 32%|############################################################1                                                                                                                               | 8/25 [00:00<00:01,  8.98it/s] 40%|##########################################################################8                                                                                                                | 10/25 [00:01<00:01,  8.75it/s] 48%|#########################################################################################7                                                                                                 | 12/25 [00:01<00:01,  9.67it/s] 56%|########################################################################################################7                                                                                  | 14/25 [00:01<00:01,  9.66it/s] 60%|################################################################################################################2                                                                          | 15/25 [00:01<00:01,  8.73it/s] 68%|###############################################################################################################################1                                                           | 17/25 [00:01<00:00,  9.95it/s] 76%|##############################################################################################################################################1                                            | 19/25 [00:02<00:00,  8.02it/s] 84%|#############################################################################################################################################################                              | 21/25 [00:02<00:00,  9.11it/s] 92%|############################################################################################################################################################################               | 23/25 [00:02<00:00,  8.93it/s] 96%|###################################################################################################################################################################################5       | 24/25 [00:02<00:00,  8.06it/s]100%|###########################################################################################################################################################################################| 25/25 [00:02<00:00,  8.98it/s]
