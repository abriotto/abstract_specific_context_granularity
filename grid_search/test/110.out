# ["--attributes=3", "--values=4", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=100", "--batch_size=64", "--learning_rate=0.001", "--hidden_size=128", "--temp_update=0.97", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6914743781089783, "acc": 0.5248648524284363, "length": 3.5496420860290527, "mode": "train", "epoch": 1}
{"loss": 0.6802363395690918, "acc": 0.5769150257110596, "length": 2.326699733734131, "mode": "test", "epoch": 1}
{"loss": 0.633135974407196, "acc": 0.6523218750953674, "length": 3.565455436706543, "mode": "train", "epoch": 2}
{"loss": 0.5399429202079773, "acc": 0.7475953698158264, "length": 4.0, "mode": "test", "epoch": 2}
{"loss": 0.4924052059650421, "acc": 0.7591192722320557, "length": 3.9493820667266846, "mode": "train", "epoch": 3}
{"loss": 0.49631252884864807, "acc": 0.7789386510848999, "length": 4.0, "mode": "test", "epoch": 3}
{"loss": 0.45514577627182007, "acc": 0.7994668483734131, "length": 3.9732472896575928, "mode": "train", "epoch": 4}
{"loss": 0.4735519587993622, "acc": 0.8030679821968079, "length": 4.0, "mode": "test", "epoch": 4}
{"loss": 0.4055750072002411, "acc": 0.8257150650024414, "length": 3.982778787612915, "mode": "train", "epoch": 5}
{"loss": 0.4341537058353424, "acc": 0.8238805532455444, "length": 4.0, "mode": "test", "epoch": 5}
{"loss": 0.35298049449920654, "acc": 0.855492889881134, "length": 3.9892544746398926, "mode": "train", "epoch": 6}
{"loss": 0.4287582337856293, "acc": 0.8296849131584167, "length": 4.0, "mode": "test", "epoch": 6}
{"loss": 0.31217554211616516, "acc": 0.873425304889679, "length": 3.9928629398345947, "mode": "train", "epoch": 7}
{"loss": 0.36690554022789, "acc": 0.8422884941101074, "length": 4.0, "mode": "test", "epoch": 7}
{"loss": 0.2822166979312897, "acc": 0.8873706459999084, "length": 3.9949147701263428, "mode": "train", "epoch": 8}
{"loss": 0.3436477482318878, "acc": 0.8599502444267273, "length": 4.0, "mode": "test", "epoch": 8}
{"loss": 0.26502537727355957, "acc": 0.8954252004623413, "length": 3.9961280822753906, "mode": "train", "epoch": 9}
{"loss": 0.32781586050987244, "acc": 0.8609452843666077, "length": 4.0, "mode": "test", "epoch": 9}
{"loss": 0.2514055669307709, "acc": 0.8995363116264343, "length": 3.997004985809326, "mode": "train", "epoch": 10}
{"loss": 0.3424668312072754, "acc": 0.8579601645469666, "length": 4.0, "mode": "test", "epoch": 10}
{"loss": 0.23982752859592438, "acc": 0.9037471413612366, "length": 3.9974234104156494, "mode": "train", "epoch": 11}
{"loss": 0.30945175886154175, "acc": 0.8793533444404602, "length": 4.0, "mode": "test", "epoch": 11}
{"loss": 0.22712545096874237, "acc": 0.9111499786376953, "length": 3.998528242111206, "mode": "train", "epoch": 12}
{"loss": 0.3047512471675873, "acc": 0.8776119351387024, "length": 4.0, "mode": "test", "epoch": 12}
{"loss": 0.2230614274740219, "acc": 0.9115150570869446, "length": 3.998838186264038, "mode": "train", "epoch": 13}
{"loss": 0.2941882908344269, "acc": 0.8847430348396301, "length": 4.0, "mode": "test", "epoch": 13}
{"loss": 0.21072150766849518, "acc": 0.9160130620002747, "length": 3.998046636581421, "mode": "train", "epoch": 14}
{"loss": 0.27284660935401917, "acc": 0.8873134851455688, "length": 4.0, "mode": "test", "epoch": 14}
{"loss": 0.20579400658607483, "acc": 0.9201797246932983, "length": 3.999191999435425, "mode": "train", "epoch": 15}
{"loss": 0.2824854850769043, "acc": 0.8826701045036316, "length": 4.0, "mode": "test", "epoch": 15}
{"loss": 0.20357495546340942, "acc": 0.9223406314849854, "length": 3.9992806911468506, "mode": "train", "epoch": 16}
{"loss": 0.2816494107246399, "acc": 0.888391375541687, "length": 4.0, "mode": "test", "epoch": 16}
{"loss": 0.20173710584640503, "acc": 0.923748791217804, "length": 3.9992077350616455, "mode": "train", "epoch": 17}
{"loss": 0.2624746263027191, "acc": 0.8984243273735046, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.18480101227760315, "acc": 0.9303427934646606, "length": 3.9996001720428467, "mode": "train", "epoch": 18}
{"loss": 0.2354855090379715, "acc": 0.9054726362228394, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.18184413015842438, "acc": 0.9314209818840027, "length": 3.999562978744507, "mode": "train", "epoch": 19}
{"loss": 0.232759028673172, "acc": 0.9072968363761902, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.17434607446193695, "acc": 0.9352712631225586, "length": 3.999687671661377, "mode": "train", "epoch": 20}
{"loss": 0.20256221294403076, "acc": 0.9216417670249939, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.1625204086303711, "acc": 0.9408064484596252, "length": 3.9997711181640625, "mode": "train", "epoch": 21}
{"loss": 0.19566857814788818, "acc": 0.9296019673347473, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.15764591097831726, "acc": 0.9424092173576355, "length": 3.999619483947754, "mode": "train", "epoch": 22}
{"loss": 0.19003808498382568, "acc": 0.9238806366920471, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.15653923153877258, "acc": 0.942104160785675, "length": 3.999659776687622, "mode": "train", "epoch": 23}
{"loss": 0.19302287697792053, "acc": 0.9310945272445679, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.14448672533035278, "acc": 0.9462587833404541, "length": 3.9998366832733154, "mode": "train", "epoch": 24}
{"loss": 0.18400289118289948, "acc": 0.9310116767883301, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.1466745138168335, "acc": 0.9486920237541199, "length": 3.9997808933258057, "mode": "train", "epoch": 25}
{"loss": 0.17040930688381195, "acc": 0.9394693374633789, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.1393924504518509, "acc": 0.9483612179756165, "length": 3.9998326301574707, "mode": "train", "epoch": 26}
{"loss": 0.1640218198299408, "acc": 0.9442787766456604, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.13321611285209656, "acc": 0.9505239129066467, "length": 3.9999349117279053, "mode": "train", "epoch": 27}
{"loss": 0.14676806330680847, "acc": 0.9506633281707764, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.1284593939781189, "acc": 0.9537891745567322, "length": 3.9999494552612305, "mode": "train", "epoch": 28}
{"loss": 0.15399502217769623, "acc": 0.946517288684845, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.1313060075044632, "acc": 0.9515197277069092, "length": 3.999936580657959, "mode": "train", "epoch": 29}
{"loss": 0.14713329076766968, "acc": 0.9499171376228333, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.12262419611215591, "acc": 0.9553365111351013, "length": 3.9998695850372314, "mode": "train", "epoch": 30}
{"loss": 0.14470817148685455, "acc": 0.9455223083496094, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.1148163378238678, "acc": 0.9581312537193298, "length": 3.9999072551727295, "mode": "train", "epoch": 31}
{"loss": 0.13562160730361938, "acc": 0.9538142085075378, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.12017989158630371, "acc": 0.956803023815155, "length": 3.999910593032837, "mode": "train", "epoch": 32}
{"loss": 0.12855613231658936, "acc": 0.9562188386917114, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.1152443066239357, "acc": 0.9600145816802979, "length": 3.9999303817749023, "mode": "train", "epoch": 33}
{"loss": 0.13128672540187836, "acc": 0.9586236476898193, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.12098511308431625, "acc": 0.956196665763855, "length": 3.9999303817749023, "mode": "train", "epoch": 34}
{"loss": 0.14014773070812225, "acc": 0.9527361989021301, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.10920719057321548, "acc": 0.9592413306236267, "length": 3.9999871253967285, "mode": "train", "epoch": 35}
{"loss": 0.12242497503757477, "acc": 0.9548922777175903, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.10842307657003403, "acc": 0.9614550471305847, "length": 3.999988555908203, "mode": "train", "epoch": 36}
{"loss": 0.13165387511253357, "acc": 0.9568822383880615, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.10914193838834763, "acc": 0.9622672200202942, "length": 3.999624013900757, "mode": "train", "epoch": 37}
{"loss": 0.1275339126586914, "acc": 0.9606963992118835, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.10390263050794601, "acc": 0.9639418125152588, "length": 3.999911308288574, "mode": "train", "epoch": 38}
{"loss": 0.11224158108234406, "acc": 0.9626035690307617, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.10471615940332413, "acc": 0.9626425504684448, "length": 3.99992036819458, "mode": "train", "epoch": 39}
{"loss": 0.1262459009885788, "acc": 0.9575456380844116, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.1124584749341011, "acc": 0.9601545929908752, "length": 3.9999923706054688, "mode": "train", "epoch": 40}
{"loss": 0.1179903894662857, "acc": 0.9626866579055786, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.09940169006586075, "acc": 0.9675151109695435, "length": 3.9999983310699463, "mode": "train", "epoch": 41}
{"loss": 0.10975698381662369, "acc": 0.9635985493659973, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.0917503759264946, "acc": 0.9697011113166809, "length": 3.9999966621398926, "mode": "train", "epoch": 42}
{"loss": 0.10527437180280685, "acc": 0.9683250784873962, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.09143345803022385, "acc": 0.9705033898353577, "length": 3.999992609024048, "mode": "train", "epoch": 43}
{"loss": 0.10313743352890015, "acc": 0.9684909582138062, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.09209796786308289, "acc": 0.9692583680152893, "length": 3.999999523162842, "mode": "train", "epoch": 44}
{"loss": 0.10960638523101807, "acc": 0.96683269739151, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.09723769873380661, "acc": 0.9683728218078613, "length": 3.9999983310699463, "mode": "train", "epoch": 45}
{"loss": 0.10640840977430344, "acc": 0.9674957990646362, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.09225967526435852, "acc": 0.9689804315567017, "length": 3.999969005584717, "mode": "train", "epoch": 46}
{"loss": 0.15858738124370575, "acc": 0.9533168077468872, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.10212225466966629, "acc": 0.9641945958137512, "length": 3.999997615814209, "mode": "train", "epoch": 47}
{"loss": 0.11764677613973618, "acc": 0.9601989984512329, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.10029829293489456, "acc": 0.9649689793586731, "length": 3.9999897480010986, "mode": "train", "epoch": 48}
{"loss": 0.1267831176519394, "acc": 0.9598671793937683, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.09798229485750198, "acc": 0.9670171141624451, "length": 3.9999983310699463, "mode": "train", "epoch": 49}
{"loss": 0.11924415081739426, "acc": 0.9621889591217041, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.10623782128095627, "acc": 0.9645820260047913, "length": 3.999999761581421, "mode": "train", "epoch": 50}
{"loss": 0.11601743847131729, "acc": 0.9612770080566406, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.09175144881010056, "acc": 0.9692031741142273, "length": 3.999999761581421, "mode": "train", "epoch": 51}
{"loss": 0.1086650863289833, "acc": 0.9653398990631104, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.08502451330423355, "acc": 0.9727448225021362, "length": 3.999997615814209, "mode": "train", "epoch": 52}
{"loss": 0.09435836225748062, "acc": 0.9692369699478149, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.07978954166173935, "acc": 0.9743754863739014, "length": 3.999948024749756, "mode": "train", "epoch": 53}
{"loss": 0.10318863391876221, "acc": 0.9680762887001038, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.08232694119215012, "acc": 0.9731876254081726, "length": 3.999998092651367, "mode": "train", "epoch": 54}
{"loss": 0.10098321735858917, "acc": 0.9715589284896851, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.08105629682540894, "acc": 0.9746540188789368, "length": 3.9999992847442627, "mode": "train", "epoch": 55}
{"loss": 0.10338371992111206, "acc": 0.9661692976951599, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.0796956717967987, "acc": 0.9747915863990784, "length": 3.9999876022338867, "mode": "train", "epoch": 56}
{"loss": 0.11137217283248901, "acc": 0.9653398990631104, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.08065099269151688, "acc": 0.9745431542396545, "length": 3.999992609024048, "mode": "train", "epoch": 57}
{"loss": 0.1007542610168457, "acc": 0.9715587496757507, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.07386941462755203, "acc": 0.97648024559021, "length": 3.9999983310699463, "mode": "train", "epoch": 58}
{"loss": 0.09338535368442535, "acc": 0.9722222089767456, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.10015107691287994, "acc": 0.9672626852989197, "length": 3.999911069869995, "mode": "train", "epoch": 59}
{"loss": 0.1443530172109604, "acc": 0.9528189897537231, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.10339664667844772, "acc": 0.9655504822731018, "length": 3.999999523162842, "mode": "train", "epoch": 60}
{"loss": 0.13129237294197083, "acc": 0.955970048904419, "length": 4.0, "mode": "test", "epoch": 60}
{"loss": 0.10157296806573868, "acc": 0.9680961966514587, "length": 3.9999988079071045, "mode": "train", "epoch": 61}
{"loss": 0.14026576280593872, "acc": 0.9554725885391235, "length": 4.0, "mode": "test", "epoch": 61}
{"loss": 0.11190269142389297, "acc": 0.9608114361763, "length": 3.9998621940612793, "mode": "train", "epoch": 62}
{"loss": 0.1484312266111374, "acc": 0.9446103572845459, "length": 4.0, "mode": "test", "epoch": 62}
{"loss": 0.11726577579975128, "acc": 0.9606518745422363, "length": 3.999974489212036, "mode": "train", "epoch": 63}
{"loss": 0.12583793699741364, "acc": 0.9563018679618835, "length": 4.0, "mode": "test", "epoch": 63}
{"loss": 0.09748692810535431, "acc": 0.9640287756919861, "length": 4.0, "mode": "train", "epoch": 64}
{"loss": 0.11223138868808746, "acc": 0.9597843289375305, "length": 4.0, "mode": "test", "epoch": 64}
{"loss": 0.08903562277555466, "acc": 0.9692306518554688, "length": 3.9999983310699463, "mode": "train", "epoch": 65}
{"loss": 0.1032164916396141, "acc": 0.9644277095794678, "length": 4.0, "mode": "test", "epoch": 65}
{"loss": 0.07775314152240753, "acc": 0.974764883518219, "length": 4.0, "mode": "train", "epoch": 66}
{"loss": 0.10791172087192535, "acc": 0.96600341796875, "length": 4.0, "mode": "test", "epoch": 66}
{"loss": 0.07731033861637115, "acc": 0.9753181338310242, "length": 3.999999523162842, "mode": "train", "epoch": 67}
{"loss": 0.13264493644237518, "acc": 0.9578772187232971, "length": 4.0, "mode": "test", "epoch": 67}
{"loss": 0.12964580953121185, "acc": 0.9572494626045227, "length": 4.0, "mode": "train", "epoch": 68}
{"loss": 0.1600649207830429, "acc": 0.9489221572875977, "length": 4.0, "mode": "test", "epoch": 68}
{"loss": 0.10627535730600357, "acc": 0.9647760391235352, "length": 4.0, "mode": "train", "epoch": 69}
{"loss": 0.12414611876010895, "acc": 0.9617745876312256, "length": 4.0, "mode": "test", "epoch": 69}
{"loss": 0.0898931697010994, "acc": 0.9713613986968994, "length": 4.0, "mode": "train", "epoch": 70}
{"loss": 0.13758985698223114, "acc": 0.9598674178123474, "length": 4.0, "mode": "test", "epoch": 70}
{"loss": 0.09529564529657364, "acc": 0.967570424079895, "length": 3.999999523162842, "mode": "train", "epoch": 71}
{"loss": 0.12407461553812027, "acc": 0.9616915583610535, "length": 4.0, "mode": "test", "epoch": 71}
{"loss": 0.091190405189991, "acc": 0.9703929424285889, "length": 3.999999761581421, "mode": "train", "epoch": 72}
{"loss": 0.11512430012226105, "acc": 0.9595355987548828, "length": 4.0, "mode": "test", "epoch": 72}
{"loss": 0.09223009645938873, "acc": 0.9693690538406372, "length": 3.999999523162842, "mode": "train", "epoch": 73}
{"loss": 0.11782906204462051, "acc": 0.9604477882385254, "length": 4.0, "mode": "test", "epoch": 73}
{"loss": 0.08409195393323898, "acc": 0.9727392196655273, "length": 3.9998810291290283, "mode": "train", "epoch": 74}
{"loss": 0.1019861027598381, "acc": 0.9669153094291687, "length": 4.0, "mode": "test", "epoch": 74}
{"loss": 0.07243334501981735, "acc": 0.9775871634483337, "length": 3.999999761581421, "mode": "train", "epoch": 75}
{"loss": 0.08974213153123856, "acc": 0.9713102579116821, "length": 4.0, "mode": "test", "epoch": 75}
{"loss": 0.07001391053199768, "acc": 0.9783341884613037, "length": 3.999999761581421, "mode": "train", "epoch": 76}
{"loss": 0.09393326193094254, "acc": 0.9703150391578674, "length": 4.0, "mode": "test", "epoch": 76}
{"loss": 0.06899381428956985, "acc": 0.9779190421104431, "length": 4.0, "mode": "train", "epoch": 77}
{"loss": 0.09055075794458389, "acc": 0.9703150391578674, "length": 4.0, "mode": "test", "epoch": 77}
{"loss": 0.06492169201374054, "acc": 0.9808522462844849, "length": 3.9999992847442627, "mode": "train", "epoch": 78}
{"loss": 0.08553770929574966, "acc": 0.9740464091300964, "length": 4.0, "mode": "test", "epoch": 78}
{"loss": 0.0654362365603447, "acc": 0.9793856143951416, "length": 3.999999761581421, "mode": "train", "epoch": 79}
{"loss": 0.09433665126562119, "acc": 0.9698174595832825, "length": 4.0, "mode": "test", "epoch": 79}
{"loss": 0.06375027447938919, "acc": 0.9792748689651489, "length": 3.999997615814209, "mode": "train", "epoch": 80}
{"loss": 0.09681929647922516, "acc": 0.97039794921875, "length": 4.0, "mode": "test", "epoch": 80}
{"loss": 0.06684193760156631, "acc": 0.9791643023490906, "length": 3.9999966621398926, "mode": "train", "epoch": 81}
{"loss": 0.09422551095485687, "acc": 0.9721392393112183, "length": 4.0, "mode": "test", "epoch": 81}
{"loss": 0.06378286331892014, "acc": 0.9810456037521362, "length": 3.9999842643737793, "mode": "train", "epoch": 82}
{"loss": 0.08196701854467392, "acc": 0.9754560589790344, "length": 4.0, "mode": "test", "epoch": 82}
{"loss": 0.06207532435655594, "acc": 0.9812949299812317, "length": 4.0, "mode": "train", "epoch": 83}
{"loss": 0.09970702975988388, "acc": 0.9706467390060425, "length": 4.0, "mode": "test", "epoch": 83}
{"loss": 0.06777084618806839, "acc": 0.9792467355728149, "length": 3.9999895095825195, "mode": "train", "epoch": 84}
{"loss": 0.08758542686700821, "acc": 0.968988299369812, "length": 4.0, "mode": "test", "epoch": 84}
{"loss": 0.05835335701704025, "acc": 0.9830935597419739, "length": 3.9999988079071045, "mode": "train", "epoch": 85}
{"loss": 0.08435880392789841, "acc": 0.9752903580665588, "length": 4.0, "mode": "test", "epoch": 85}
{"loss": 0.05481826514005661, "acc": 0.9837022423744202, "length": 4.0, "mode": "train", "epoch": 86}
{"loss": 0.0761580765247345, "acc": 0.9776121377944946, "length": 4.0, "mode": "test", "epoch": 86}
{"loss": 0.05737224221229553, "acc": 0.9835084676742554, "length": 3.999999761581421, "mode": "train", "epoch": 87}
{"loss": 0.08428463339805603, "acc": 0.9733831882476807, "length": 4.0, "mode": "test", "epoch": 87}
{"loss": 0.05817057192325592, "acc": 0.982789158821106, "length": 4.0, "mode": "train", "epoch": 88}
{"loss": 0.09146171808242798, "acc": 0.9726366996765137, "length": 4.0, "mode": "test", "epoch": 88}
{"loss": 0.07031324505805969, "acc": 0.9779731035232544, "length": 3.999967336654663, "mode": "train", "epoch": 89}
{"loss": 0.12468623369932175, "acc": 0.966335117816925, "length": 4.0, "mode": "test", "epoch": 89}
{"loss": 0.08577583730220795, "acc": 0.9714995622634888, "length": 3.999999761581421, "mode": "train", "epoch": 90}
{"loss": 0.10652700811624527, "acc": 0.9667496085166931, "length": 4.0, "mode": "test", "epoch": 90}
{"loss": 0.06902176886796951, "acc": 0.9760655164718628, "length": 4.0, "mode": "train", "epoch": 91}
{"loss": 0.11089954525232315, "acc": 0.9649255275726318, "length": 4.0, "mode": "test", "epoch": 91}
{"loss": 0.07900336384773254, "acc": 0.9734417796134949, "length": 3.999025344848633, "mode": "train", "epoch": 92}
{"loss": 0.10987064987421036, "acc": 0.9660031199455261, "length": 4.0, "mode": "test", "epoch": 92}
{"loss": 0.06998525559902191, "acc": 0.9772273898124695, "length": 3.999999523162842, "mode": "train", "epoch": 93}
{"loss": 0.10614544153213501, "acc": 0.9651740193367004, "length": 4.0, "mode": "test", "epoch": 93}
{"loss": 0.06888207048177719, "acc": 0.9773104786872864, "length": 3.999997615814209, "mode": "train", "epoch": 94}
{"loss": 0.11854376643896103, "acc": 0.9631010890007019, "length": 4.0, "mode": "test", "epoch": 94}
{"loss": 0.08488090336322784, "acc": 0.96939617395401, "length": 3.999969959259033, "mode": "train", "epoch": 95}
{"loss": 0.12614458799362183, "acc": 0.9620231986045837, "length": 4.0, "mode": "test", "epoch": 95}
{"loss": 0.08396409451961517, "acc": 0.9724957942962646, "length": 3.999999761581421, "mode": "train", "epoch": 96}
{"loss": 0.09578201174736023, "acc": 0.971310019493103, "length": 4.0, "mode": "test", "epoch": 96}
{"loss": 0.07416520267724991, "acc": 0.9749031662940979, "length": 3.999999523162842, "mode": "train", "epoch": 97}
{"loss": 0.09157925844192505, "acc": 0.9727197289466858, "length": 4.0, "mode": "test", "epoch": 97}
{"loss": 0.063905268907547, "acc": 0.9784726500511169, "length": 3.999999761581421, "mode": "train", "epoch": 98}
{"loss": 0.08515425771474838, "acc": 0.9737147092819214, "length": 4.0, "mode": "test", "epoch": 98}
{"loss": 0.06042604520916939, "acc": 0.9809905290603638, "length": 3.999999523162842, "mode": "train", "epoch": 99}
{"loss": 0.08175992965698242, "acc": 0.9742124080657959, "length": 4.0, "mode": "test", "epoch": 99}
{"loss": 0.061194851994514465, "acc": 0.9813225865364075, "length": 4.0, "mode": "train", "epoch": 100}
{"loss": 0.10338368266820908, "acc": 0.9683250784873962, "length": 4.0, "mode": "test", "epoch": 100}
test accuracy: 0.7785592675209045
